keps/976-plain-pods/README.md

```diff
@@ -425,6 +425,7 @@ order to create a Workload object.
 To fully identify the group of pods, the pods need the following:
 - the label `kueue.x-k8s.io/pod-group-name`, as a unique identifier for the group. This should
   be a valid CRD name.
+- The label `kueue.x-k8s.io/pod-group-pod-index`, to indicate an index of a pod within the group
```

Comment:
Is this a required label for PodGroup? Or optional label?

Reply:
Additionally, could you update the reference doc?
https://kueue.sigs.k8s.io/docs/reference/labels-and-annotations/

Reply:
Optional, I've clarified that and updated the reference doc